src/partials/configure-github-app.md

```diff
@@ -0,0 +1,110 @@
+# Configure GitHub app {% #git %}
+
+Appwrite supports automatic deployments through Git integration. In order for Appwrite to access your repos, you must [create a GitHub app](https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps) to enable this integration. The GitHub app requires the following configurations.
+
+For automatic Git deployment to work, Appwrite needs to receive communication from GitHub, this means your Appwrite project must be accessible on the internet. If you're running on `localhost`, you need to run a proxy like [ngrok](https://ngrok.com/).
+
+## GitHub App name {% #github-app-name %}
+
+The GitHub App name will be displayed when connecting Appwrite to GitHub. In addition, this name will be transformed to a slug ("My GitHub App" will become "my-github-app") and appear in the URL when accessing your app in GitHub. This slug should be set as the `_APP_VCS_GITHUB_APP_NAME` environment variable in Appwrite.
+
+## Homepage URL {% #github-homepage-url %}
+
+The homepage URL will appear when looking at the public page of your app. It can be any URL you'd like.
+
+## Callback URL {% #callback %}
+
+GitHub will use **callback URLs** to redirect users back to Appwrite. Set these callback URLs under **Identifying and authorizing users** in the same order as listed below.
+
+| URLs                                                                          |
+| ----------------------------------------------------------------------------- |
+| `https://[HOSTNAME_OR_IP]/v1/vcs/github/callback`                             |
+| `https://[HOSTNAME_OR_IP]/v1/account/sessions/oauth2/callback/github/console` |
```

Comment:
Let's use angle brackets instead of square brackets for all placeholders.

Reply:
Ok